crates/eww/src/widgets/systray.rs

```diff
@@ -188,3 +189,18 @@ impl Item {
         }
     }
 }
+
+async fn load_icon_for_item(icon: &gtk::Image, item: &notifier_host::Item, size: i32, scale: i32) {
+    if let Some(pixbuf) = item.icon(size, scale).await {
+        let surface = unsafe {
+            // gtk::Image will destroy the surface on drop
```

Comment:
can you link documentation that states this?

Reply:
oops my comment was kinda wrong. I meant gtk::cairo::Surface would free the C pointer on drop https://github.com/gtk-rs/gtk-rs-core/blob/fe1a41ce63ef71dc5f8ec7eb7f342a45ca025277/cairo/src/surface.rs#L353
For the Surface struct itself, I guess we don't need to take extra care about it?